manifests/metallb.yaml

```diff
@@ -137,6 +137,9 @@ spec:
         prometheus.io/scrape: "true"
         prometheus.io/port: "7472"
     spec:
+      tolerations:
```

Comment:
Can you make this change in the helm chart (under `helm-chart/templates/speaker.yaml`), and then run `make manifest` to regenerate this file?

The authoritative copy of the manifests is the helm chart, because we can generate the "flat" manifest from the helm chart, but not vice versa.

Thank you!

Reply:
@danderson yep, I already found it after checking the cirleci logs, done :-)